src/frontend/web_application/src/scenes/ContactBook/presenter.jsx

```diff
@@ -98,20 +88,22 @@ class ContactBook extends Component {
       });
     };
 
-    /* const handleSortViewChange = (event) => {
+    const handleSortViewChange = (event) => {
       this.setState({
         sortView: event.target.value,
       });
-    };*/
+    };
 
     const { contacts, isFetching, hasMore, __ } = this.props;
 
+    const tags = [].concat(...contacts.map(contact => contact.tags));
```

Comment:
@iamdey @ziir finally, don't need tags in state, so I just passed it in render().